__mocks__/@grafana/ui.tsx

```diff
@@ -47,7 +47,7 @@ const MultiSelect = forwardRef((props, ref) => <Select {...props} ref={ref} mult
 MultiSelect.displayName = 'MultiSelect';
 
 // Using this specifically for Probe filter at the moment
-const mockProbeOptions = [{ label: 'Chicago', value: 22 }];
+const mockProbeOptions = [{ label: `tacos`, value: 1 }];
```

Comment:
This should reference PRIVATE_PROBE.name but it can't compile correctly. This will be removed and mocked correctly once the react-query PR goes in.